package.json

```diff
@@ -21,6 +26,11 @@
     "jquery": "^3.2.1"
   },
   "devDependencies": {
-    "live-server": "^1.2.0"
+    "autoprefixer": "^7.1.1",
+    "chokidar-cli": "^1.2.0",
+    "live-server": "^1.2.0",
+    "node-sass-chokidar": "0.0.2",
```

Comment:
`node-sass-chokidar` Package for compiling sass files. "Chokidar" (meaning watcher / gatekeeper in Hindi) is a package for watching files for changes. So this package will watch or .scss files for changes, and then compile the Sass into CSS.